test/sol6/kyberFeeHandler.js

```diff
@@ -1562,6 +1566,107 @@ contract('KyberFeeHandler', function(accounts) {
                 Helper.assertEqual(0, await feeHandler.getLatestSanityRate());
             });
         });
+
+        describe("reentrancy lead to inaccurate event when claiming fee", async() =>{
```

Comment:
I would write

test claim fee functions (claimRebate etc..) in fee handler are blocking re entrance calls.

Reply:
updated